x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/create_rules.ts

```diff
@@ -82,8 +84,8 @@ export const createRules = ({
       },
       schedule: { interval },
       enabled,
-      actions: [], // TODO: Create and add actions here once we have email, etc...
-      throttle: null,
+      actions: actions ?? [],
```

Comment:
The `createRulesSchema` validation should set this default, right? Do we need the coalescing here?

Reply:
This might be because of the tech debt with some of the TypeScript "soup" issues that are from me when sharing types between patch and create here:

```ts
x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/types.ts
```

If that's true, and it's messy to correct this at this point without large refactoring you can add a comment like so:
```ts
actions ?? [], // FUNFACT: This is really never undefined here but since types are over shared between patch and create it exists here.
```

And we can clean this up down the road.